objectstore/objectstore.go

```diff
@@ -1,11 +1,14 @@
 package objectstore
 
 import (
+	"errors"
 	"io"
 
 	"github.com/fastly/compute-sdk-go/internal/abi/fastly"
 )
 
+var ErrNotFound = errors.New("objectstore: key not found")
```

Comment:
What does `Open` return if a store can't be found?  Should it be unified with this value?  Or should this value be called `ErrKeyNotFound` instead?

Reply:
I believe you get a `FastlyError` (forwarded from upstream) with `Status`
https://github.com/fastly/compute-sdk-go/blob/main/internal/abi/fastly/types.go#L46

Reply:
A bigger question is maybe: should we be exposing `FastlyError` out of `internal` at all, or should we capture and have module-specific errors?

Reply:
Yeah, I was wondering about that.  I'll open a bug.